source/catalog.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html class="catalog" lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+<body class="catalog__body">
+  <header class="main-header">
+    <a class="main-header__logo logo" href="index.html">
+      <img class="logo__image logo-image" src="img/logo-mobile.svg" height="38" width="33" alt="Надпись Кот Энерджи.">
+      <img class="logo__text logo-text" src="img/logo-text.svg" height="18" width="101" alt="Логотип Кот Энерджи.">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">Открыть меню</span>
+      </button>
+      <ul class="main-nav__list">
+        <li class="main-nav__item">
+          <a href="index.html">Главная</a>
+        </li>
+        <li class="main-nav__item main-nav__item--current">
+          <a>Каталог продукции</a>
+        </li>
+        <li class="main-nav__item">
+          <a href="form.html">Подбор программы</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main-catalog">
+    <div class="main-catalog__container">
+      <h1 class="main-catalog__title">Каталог продукции</h1>
+      <ul class="main-catalog__list">
+        <li class="main-catalog__item product-card">
+          <img class="product-card_image" src="img/catalog/cat-energipro-500-chicken.png"
```

Comment:
Есть ошибки именования классов по БЭМ. Проверить можно [здесь](https://yoksel.github.io/html-tree/). В данном случае отсутствует одно нижнее подчёркивание:
```suggestion
          <img class="product-card__image" src="img/catalog/cat-energipro-500-chicken.png"
```